docs/man/monerod.1

```diff
@@ -0,0 +1,5343 @@
+.TH MONEROD "1" "AUGUST 2024" "Fluorine Fermi 0.18.3.3" "MONEROD MANUAL"
+.SH NAME
+monerod \- Connects you to Monero network
```

Comment:
Daemon descriptions in man pages tend to be a bit more precise, I might suggest:
```
Monero p2p network daemon
```